.github/workflows/build-master.yml

```diff
@@ -39,6 +40,11 @@ jobs:
                     name: Code Coverage JSON
                     path: graphql-ballerina/target/report/test_results.json
                     if-no-files-found: ignore
+            -   name: Generate CodeCov Report
+                uses: codecov/codecov-action@v1
+                with:
+                    token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Do we need the code cov token?

Reply:
Found in Documentation that its not need for public repos. Will test and change